ixmp/core.py

```diff
@@ -490,10 +502,10 @@ def add_timeseries(self, df, meta=False):
         # Ensure consistent column names
         df = to_iamc_template(df)
 
-        if 'time' not in df.columns:
-            df['time'] = 'Year'
+        if 'subannual' not in df.columns:
```

Comment:
setting the default should happen inside the Java core, in my opinion

Reply:
it is done like this for simplicity - java already set default value if passed null value as timeslice name. Feel free to rewrite this method to incorporate your suggested change.

Reply:
please have a look on updated implementation